src/generators/addon-verify/utils/generateFileList.mjs

```diff
@@ -0,0 +1,58 @@
+/**
+ * Updates JavaScript files with correct require paths for the build tree.
+ *
+ * @param {string} content Original code
+ * @returns {string}
+ */
+const updateJsRequirePaths = content => {
+  return `'use strict';
+const common = require('../../common');
```

Comment:
Can you use `dedent` so that the code is within the same padding level as the function?